subjects/luhn_algorithm/README.md

```diff
@@ -0,0 +1,62 @@
+## luhn_algorithm
+
+### Instructions
+
+Create a function which checks if a number is valid per the Luhn formula.
+
+The function will receive a string and return a boolean.
```

Comment:
This section is a bit confusing maybe we can separate it like this:

```
The function will receive a string as its input and return a boolean value and you'll have to take into account the following:

**Invalid Inputs:** It categorizes an input as invalid if it is an empty string or a number with only one digit.

**Handling Spaces:** While spaces within the input are allowed, they are automatically removed during the calculation process, ensuring they do not impact the result.
```

Reply:
done